content/600-about/200-prisma-docs/20-style-guide/05-prisma-product-names.mdx

```diff
@@ -5,17 +5,48 @@ metaDescription: 'This section of the style guide provides guidelines on how to
 tocDepth: 3
 ---
 
-## Use the following forms
+Use the official Prisma product names and feature names listed below in documentation. Do not abbreviate any of these names, except where noted.
+
+## Product names
 
 - The Prisma Data Platform (or just "PDP" - no definite article - after the first mention)
 - Prisma Client (not any variation of "the Client")
 - Prisma Schema
 - Prisma Migrate
 - Prisma CLI
 - Prisma Studio
+
+## Feature names
+
 - Data Proxy (_not_ Prisma Data Proxy)
```

Comment:
```suggestion
- The Data Proxy (_not_ Prisma Data Proxy)
```

See my comment below about definite articles for feature names. We might want to add definite articles to some/all of the other items in this list.

Reply:
I want to only list feature names and avoid the general rule that we always use "the" in front. I've noted in the rule notes below to use that "when appropriate". I don't think it always makes sense to use "the" for these.